react/week2/todo-list/src/todo.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react"
+
+const Timer =()=>{
+    const [timer, setTimer]= useState(0);
+    useEffect(()=>{
+       const interval= setInterval(()=>{
+           setTimer(timer => timer + 1);
+       },1000);
+       return()=> clearInterval(interval);
+    },[])
+    return(
+        <div>
+            <p>You have used {timer} seconds on this website </p>
+        </div>
+           )
+}
+
+const Todo= ({description , id, delTodo, check,toggleCheckbox})=>{
+    //const[check,setCheckbox] = useState(false);
+return(
+      <div className='todoItems'>
+        <h2 className={check ?'checkbox': '' }>{description}</h2>
+        <input className='checkbox' type="checkbox" checked={check} onChange={()=>toggleCheckbox(id) } >
+        </input>
```

Comment:
There is no need of passing check and toggleCheckbox in ToDo. You were thinking right but u did not use state. This will simplify your code. Also remove this function in other place also.
const Todo= ({description , id, delTodo})=>{
const[check,setCheckbox] = useState(false);
<input  type="checkbox" checked={check} onChange={()=>setCheckbox(!check) } >